lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -96,11 +96,11 @@ impl Client {
 
         let mut rdpdr = Rdpdr::new(
             Box::new(TeleportRdpdrBackend::new(
-                SCARD_DEVICE_ID,
                 client_handle.clone(),
                 params.cert_der,
                 params.key_der,
                 pin,
+                cgo_handle,
```

Comment:
Should we just call this `handle` or `client_handle`?

Reply:
There's a separate `client_handle` above, so I call it `cgo_handle` to distinguish from that.